### ISSUE TITLE
Support Node.js v5 and npm v3, drop support for Node.js v0.12

### ISSUE THREAD

**Issue by arithmetric** [phase2/grunt-drupal-tasks#241]
For the 1.0 release, Grunt Drupal Tasks should drop support for Node.js v0.12, making v4 the oldest version supported. Additionally any compatibility breaking changes needed to support Node.js v5 can be made.

Currently, Travis CI builds show an error during build when the clean:temp task is run for the second time. The following error breaks the default build:

```
Running "clean:temp" (clean) task
Warning: Cannot delete files outside the current working directory. Use --force to continue.
Aborted due to warnings.
```

This error is discussed in the grunt-contrib-clean issue queue, and there is a fix in Node.js:
https://github.com/gruntjs/grunt-contrib-clean/issues/85

Related is the update to Zombie v4: #223

**Comment by arithmetric** [comment]
The fix in Node.js mentioned above should be released with 5.7.1.

**Comment by arithmetric** [comment]
Node.js 5.7.1 is released, and this has resolved the previously reported error.

Now, Node.js v5 only causes breaking incompatibilities with the old version of Zombie.js that is included.

**Comment by arithmetric** [comment]
This has been merged into the v1.0.0-pre branch.
